examples/imageviewer/webApp/src/jsMain/resources/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
```suggestion
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

It was missing in the original example - I agree, but It would work better in mobile web with this :)